profiles/package.mask

```diff
@@ -1,3 +1,7 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (20 Dec 2020)
+# Masked for removal, as there is a newer version available
+=dev-haskell/cabal-install-3.0.0.0
```

Comment:
I suggest not to mask otherwise working package and give people chance to install it if they need to roll back for a short while.

Reply:
Took out all masks.